nomenclature/codelist.py

```diff
@@ -542,3 +541,41 @@ def from_directory(cls, name: str, path: Path, file_glob_pattern: str = "**/*"):
             mapping[code.name] = code
 
         return cls(name=name, mapping=mapping)
+
```

Comment:
`hierarchy` would need to be a property if it should be called like this `RegionCodeList.hierarchy` as mentioned in the error message in line 579:

```suggestion
    @property
```

As it currently stands you'd have to use `RegionCodeList.hierarchy()`.

Reply:
I think that a property is more appropriate given the rest of the API...

Reply:
e.g., you can do 
```python
RegionCodeList["Some Region"].hierarchy
```
gives the hierarchy for a particular region
```python
RegionCodeList.hierarchy
```
gives the list of all hierarchy levels